mRemoteV1/Resources/Language/Language.resx

```diff
@@ -2794,13 +2793,19 @@ Development Channel includes Alphas, Betas &amp; Release Candidates.</value>
     <value>Environment</value>
   </data>
   <data name="EncryptionTestResultMessage" xml:space="preserve">
-      <value>Encrypting {0} entries using {1}/{2} and {3} iterations took {4} seconds.</value>
+    <value>Encrypting {0} entries using {1}/{2} and {3} iterations took {4} seconds.</value>
   </data>
   <data name="EncryptionTest" xml:space="preserve">
-      <value>Encryption Test</value>
+    <value>Encryption Test</value>
   </data>
   <data name="TestSettings" xml:space="preserve">
-      <value>Test Settings</value>
+    <value>Test Settings</value>
+  </data>
+  <data name="strPropertyDescriptionRdpProtocolVersion" xml:space="preserve">
+    <value>Sets the version of the RDP protocol to use when opening connections.</value>
```

Comment:
Since the R ind RDP already stands for protocol it feels wierd to read 'RDP protocol'. I would change this sentence to 

> Sets the RDP version to use when opening connections